build/vscode-languageserver-files-stub.js

```diff
@@ -0,0 +1,64 @@
+// This is a stub to provide only the part of node_modules/vscode-languageserver/lib/files.js that is used by dockerfile-language-server-node
```

Comment:
This is the exact same solution one of the built-in vscode extensions uses (https://github.com/Microsoft/vscode/blob/f9f38b2bbf5d5049cad387b97b8fa85e98c6d3c4/extensions/html-language-features/server/extension.webpack.config.js#L27-L30).  In a way all the fixes are kind of hackish, since the code wasn't intended to be run in a single file.  We could ask them to fix, but really what's there to fix?  They're not doing anything wrong.

Here was their response about this:
> Regarding (1-2): if your server is not trying to load a npm module relatively to the workspace folder during runtime using Files.resolveModule then the trick John recommended works for you as well. If you have such a need migrate to the new Files.resolve method which even works when using web pack and still use the trick to calm the warning. At some point I will remove Files.resolveModule but this needs a new major release number.

Personally I think this is not worth the time to try to fix since that's only used there and the code is obsolete.

Reply:
Okay no need to file an issue. Maybe add a comment with the above link to say where you got it from

(Just in case they change it in the future, we could just copy their changes)

Reply:
Although I think the link is only helpful if you actually copied a decent amount from there. It's not clear to me how much you copied and how much you did yourself.

Reply:
Done